samples/CommunityToolkit.Maui.Sample/Pages/Converters/IsNotNullConverterPage.xaml

```diff
@@ -0,0 +1,57 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<pages:BasePage
+    x:Class="CommunityToolkit.Maui.Sample.Pages.Converters.IsNotNullConverterPage"
+    xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+    xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+    xmlns:mct="http://schemas.microsoft.com/dotnet/2022/maui/toolkit"
+    xmlns:pages="clr-namespace:CommunityToolkit.Maui.Sample.Pages"
+    xmlns:vm="clr-namespace:CommunityToolkit.Maui.Sample.ViewModels.Converters"
+    x:DataType="vm:IsNotNullConverterViewModel"
+    x:TypeArguments="vm:IsNotNullConverterViewModel">
+
+    <pages:BasePage.Resources>
+        <ResourceDictionary>
+            <mct:IsNotNullConverter x:Key="IsNotNullConverter" />
+        </ResourceDictionary>
+    </pages:BasePage.Resources>
+    <pages:BasePage.Content>
+        <Grid
+            Padding="10,10"
+            HorizontalOptions="Fill"
+            RowDefinitions="Auto, Auto, Auto, Auto, Auto"
+            VerticalOptions="Fill">
+            <Label
+                Grid.Row="0"
+                Text="The IsNotNullConverter is a converter that allows users to convert an incoming object? to a bool. If the incoming object? is not null, it will be converted to true. Any other incoming value will be converted to false."
+                TextColor="{StaticResource NormalLabelTextColor}" />
+            <HorizontalStackLayout Grid.Row="1">
```

Comment:
The idea to use Grid was to remove HorizontalStackLayout in VerticalStackLayout

Reply:
@VladislavAntonyuk should i change it back to VerticalStackLayout?

Reply:
We don't want to nest a layout inside of a layout:

> ...don't attempt to reproduce the appearance of a specific layout by using combinations of other layouts, as this results in unnecessary layout calculations being performed. For example, don't attempt to reproduce a Grid layout by using a combination of StackLayout instances

https://docs.microsoft.com/xamarin/xamarin-forms/deploy-test/performance#choose-the-correct-layout?WT.mc_id=mobile-0000-bramin

Reply:
No, let's use 1 Grid Control instead of HorizontalStackLayout in VerticalStackLayout. So we use one control instead of multiple.
Like this:
![image](https://user-images.githubusercontent.com/33021114/163725942-17f6b277-dca3-48ab-922a-ffdbbb3f0378.png)

Reply:
@VladislavAntonyuk ahh ok, is the last commit ok? will update the other page if it is

Reply:
LGTM